requirements-test.txt

```diff
@@ -0,0 +1,3 @@
+mock
```

Comment:
Turns out we need `tox>=1.8.0` for the expansion syntax in the tox.ini. I had an old version of tox locally and it wasn't working.

https://testrun.org/tox/latest/changelog.html#id2